main.js

```diff
@@ -126,55 +117,64 @@ function resetButtons() {
-  currentActivity.completed = true;
-  pastActivities.push(currentActivity)
-  currentActivity.saveToStorage();//testing this
-  cardContainer.innerHTML +=
-    `<article>
-      <div class="card-section">
-        <p class="card-title">${currentActivity.category}</p>
-        <p class="card-time">${currentActivity.minutes} MIN ${currentActivity.seconds} SECONDS</p>
-        <p class="card-description">${currentActivity.description}</p>
-      </div>
-      <div class="card-section">
-      <button class="card-category-indicator ${color}"type="button" name="button"></button>
-      </div>
-    </article>`
+  pastActivities.unshift(currentActivity)
+  currentActivity.saveToStorage();
+  loadPastCards()
 };
 
-
-function windowLoad() {
-  var page = localStorage.getItem('savedArray')
-  var info = JSON.parse(page)
-  pastActivities = info
-
-  loadCard(pastActivities)
-}
-
-function loadCard(thing) {
-  for (var i = 0; i < thing.length; i++) {
-    var color = thing[i].category.toLowerCase()
+function loadPastCards() {
+  cardContainer.innerHTML = ""
+  for (var i = 0; i < pastActivities.length; i++) {
     cardContainer.innerHTML +=
       `<article>
       <div class="card-section">
-        <p class="card-title">${thing[i].category}</p>
-        <p class="card-time">${thing[i].minutes} MIN ${thing[i].seconds} SECONDS</p>
-        <p class="card-description">${thing[i].description}</p>
+        <p class="card-title">${pastActivities[i].category}</p>
+        <p class="card-time">${pastActivities[i].minutes} MIN ${pastActivities[i].seconds} SECONDS</p>
+        <p class="card-description">${pastActivities[i].description}</p>
       </div>
       <div class="card-section">
-      <button class="card-category-indicator ${color}"type="button" name="button"></button>
+      <button class="card-category-indicator ${pastActivities[i].category.toLowerCase()}"></button>
       </div>
     </article>`
-  }
-}
+  };
+};
+
+function retrievePastActivities() {
+  var page = localStorage.getItem('savedArray');
+  var info = JSON.parse(page);
+  pastActivities = info;
+  loadPastCards();
+};
 
 function displayLocalStorage() {
   if (localStorage.length === 0) {
-    removeClass(defaultMessage)
-    return
+    removeClass(defaultMessage);
+    return;
   }
-    windowLoad()
+    retrievePastActivities();
+};
+
+
+function displayTime(minutes, seconds) {
+  startButton.disabled = true
+  var countDownTimer = setInterval(function() {
+      if (seconds === 0 && minutes === 0) {
+        startButton.innerText = "COMPLETE!";
+        removeClass(logButton);
+        currentActivity.markComplete()
+        clearInterval(countDownTimer)
+      } else if (seconds === 0) {
+        minutes--
+        seconds = 59
+      } else {
+        seconds--
+      }
+      fixTime(minutes, seconds)
+    }, 1000);
 }
+
+function fixTime(minutes, seconds) {
+  timer.innerText = `${minutes.toString().padStart(2, '0')}:${seconds.toString().padStart(2 , '0')}`
+}
```

Comment:
Works as expected and removes the same reassignment line being used in multiple functions previously.